### ISSUE TITLE
[BUG][Python-Flask] Primitive type bytearray deserialization missing in util.py.

### ISSUE THREAD

**Issue by dannmartens** [OpenAPITools/openapi-generator#2724]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
Yes.
- [x] Have you validated the input using an OpenAPI validator ([example]
(https://apidevtools.org/swagger-parser/online/))?
- [x] What's the version of OpenAPI Generator used?
HEAD of current master branch.
- [x] Have you search for related issues/PRs?
Yes.
- [x] What's the actual output vs expected output?
```python
  File "/home/ubuntu/dvm-rest-api/dvmapi/util.py", line 111, in deserialize_model
    setattr(instance, attr, _deserialize(value, attr_type))
  File "/home/ubuntu/dvm-rest-api/dvmapi/util.py", line 32, in _deserialize
    return deserialize_model(data, klass)
  File "/home/ubuntu/dvm-rest-api/dvmapi/util.py", line 103, in deserialize_model
    if not instance.openapi_types:
AttributeError: 'bytearray' object has no attribute 'openapi_types'
```

##### Description

When an object property is specified as type:string, format:byte, no code is generated to support deserialization.

##### openapi-generator version

HEAD of current master branch -> 4.0.0 - no regression.

##### OpenAPI declaration file content or url

```yaml
openapi: 3.0.1
info:
  title: Flask Generator Issue
  description: This API provides an example of the problem with util.py generated but the Flask generator, for deserializing supported primitive types.
  version: 0.1.0
servers:
- url: http://localhost:8080
paths:
  /examples:
    get:
      summary: Example GET.
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Example'
components:
  schemas:
    Example:
      required:
      - data 
      properties:      
        data:
          type: string
          description: Base64 representation of data.
          format: byte
```

##### Steps to reproduce

Inspect util.py, there is no mention of the primitive type 'bytearray'.
Inspect models/example.py, the data property is typed 'bytearray'.

```python
        self.openapi_types = {
            'data': bytearray
        }
```

##### Suggest a fix

Update template to generate util.py to include 'bytearray' at

```python
    if klass in six.integer_types or klass in (float, str, bool):
        return _deserialize_primitive(data, klass)
```

**Comment by wing328** [comment]
@dannmartens your suggested fix sounds good to me. Please file a PR so that the CIs can verify the fix. Thanks!

**Comment by dannmartens** [comment]
Will do.

It's working on our side, already.

I have a budget for this, so very happy that I can be of assistance.

You're welcome!

**Comment by Shogoki** [comment]
Hi, Is anybody still working on a PR for that?
I just stumbled across the same thing. @dannmartens fix is working fine. 
Are you still working on a PR for that? otherwise, i would offer to take a look into it :-)

**Comment by dannmartens** [comment]
Hi there @Shogoki ,

Unfortunately, I have been collecting other issues as well, specifically with the Python Flask server stubs, next to some core wrinkles we've come across. 

Delivering our project takes precedence over contributing back, right now, so I will not be able to send a PR in the next week(s). Still struggling after our migration to 4.0.x stable, especially with the impact of pythonic_params=true. :(

**Comment by wing328** [comment]
Closed via #3642
